tsconfig.json

```diff
@@ -16,7 +16,8 @@
     "experimentalDecorators": true /* Enables experimental support for ES7 decorators. */,
     "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */,
     "resolveJsonModule": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "skipLibCheck": true
```

Comment:
had to turn this one because of some conflicts in dependencies types declarations. `@types/eslint` and `@tyes/eslint-scope` had a definition for the same type, and tsc was complaining